content/ja/handbook/marketing/blog/index.md

```
@@ -0,0 +1,381 @@
+[MR への Mermaid チャートの埋め込み方法](https://gitlab.com/gitlab-com/marketing/digital-experience/gitlab-blog/-/merge_requests/211#usage-notes)。チャートが正しくレンダリングされない可能性のあるニュアンスがあるため、お読みください。
+
+#### 画像の準備
+
+- オリジナルのカバー画像を作成する場合、すべてのディスプレイで最適な品質を得るためにサイズは 1800px × 945px にしてください。
+- すべての画像は 1MB 未満を目指してください。JPEG は PNG よりも小さい傾向があるため、可能な場合は JPEG を使用してください。
+- Preview でリサイズするには、`ツール`、`サイズを調整`に進み、`解像度`フィールドのエントリを調整します。Preview は `OK` をクリックして確認する前に、結果として得られる画像サイズを推定します。
+- すべての画像は同じ幅に保ってください。
+
+#### スクリーンショット
+
+技術 / チュートリアル投稿の場合は、[コードブロック](#adding-code-blocks)またはスクリーンショットで例を図示してください。例に一貫性を持たせてください。例えば、汎用 URL `domain.com` を使用してステップを説明する場合は、投稿全体を通じて `domain.com` を一貫して使用してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)